common/script/index.coffee

```diff
@@ -1740,14 +1744,22 @@ api.wrap = (user, main=true) ->
           dailyChecked += 1
         else
           # dailys repeat, so need to calculate how many they've missed according to their own schedule
+          console.log("task: " + task.text) # XXX Alys to remove these
           scheduleMisses = 0
+          keepGoing = true # XXX Alys to improve how we break out of _.times
           _.times daysMissed, (n) ->
-            thatDay = moment(now).subtract({days: n + 1})
-            if api.shouldDo(thatDay.toDate(), task, user.preferences)
-              scheduleMisses++
-              if user.stats.buffs.stealth
-                user.stats.buffs.stealth--
-                EvadeTask++
+            if keepGoing
+              console.log("n: " + n)
+              thatDay = moment(now).subtract({days: n + 1})
+              if api.shouldDo(thatDay.toDate(), task, user.preferences)
+                console.log("damage")
+                scheduleMisses++
+                if user.stats.buffs.stealth
+                  user.stats.buffs.stealth--
+                  EvadeTask++
+                if multiDaysCountAsOneDay
+                  console.log("end")
+                  keepGoing = false
```

Comment:
It's impossible due to the implementation of the method. I'd strongly suggest using traditional for-loops where possible. Additionally, there's a good chance that they'll be faster than alternatives.